api/src/organizational-entities/domain/errors.js

```diff
@@ -20,6 +20,13 @@ class ArchiveCertificationCentersInBatchError extends DomainError {
   }
 }
 
+class ArchiveOrganizationsInBatchError extends DomainError {
+  constructor({ meta } = {}) {
+    super('archive oganizations in batch error', 'ARCHIVE_ORGANIZATIONS_IN_BATCH_ERROR');
```

Comment:
Rien d'important, mais le message passé en argument au constructeur doit ressembler à un message qui va s'afficher dans la console 😃 On fait commencer ces messages par une majuscule. Voici une proposition :
```suggestion
    super('Error during batch archive of organizations', 'ARCHIVE_ORGANIZATIONS_IN_BATCH_ERROR');
```